Loop/Managers/DeviceDataManager.swift

```diff
@@ -147,7 +147,8 @@ final class DeviceDataManager {
 
         let fileManager = FileManager.default
         let documentsDirectory = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first!
-        deviceLog = PersistentDeviceLog(storageFile: documentsDirectory.appendingPathComponent("DeviceLog.sqlite"), maxEntryAge: localCacheDuration)
+        let deviceLogDirectory = documentsDirectory.appendingPathComponent("DeviceLog")
+        deviceLog = PersistentDeviceLog(storageFile: deviceLogDirectory.appendingPathComponent("Storage.sqlite"), maxEntryAge: localCacheDuration)
```

Comment:
After this change goes in, doing an _Issue Report_ won't return any (old) device logs.  (That is, this effectively starts the logging over).   Is that ok?

Reply:
For Tidepool, yes, since we do not have any live users. For DIY, this is the way it has always been, since the change was made before any DIY code was released.